test/logs.js

```diff
@@ -272,12 +285,14 @@ QUnit.test( module2Test4.name, function( assert ) {
 
 	assert.deepEqual( testDoneContext, {
 		assertions: [],
-		module: module2Context.name,
-		name: module2Test3.name,
+		suiteName: module2Context.name,
+		testName: module2Test3.testName,
+		status: "skipped",
```

Comment:
this is a big issue for implementing the events emitter: we can't simply remove the other properties, that would represent a breaking change and would require a major update. 

In our api development process, we want to give a reasonable time before any major update to allow users to migrate from the old to the new API, otherwise upgrading a major version would be very hard in many suites.

I need to check this EventEmiter api and maybe releasing it with the new properties coming on a follow-up PR might be easier. That's also gonna need to rebased after some changes happening right now at the current code.